roles/openshift_examples/defaults/main.yml

```diff
@@ -6,7 +6,7 @@ openshift_examples_load_db_templates: true
 openshift_examples_load_xpaas: "{{ openshift_deployment_type == 'enterprise' }}"
 openshift_examples_load_quickstarts: true
 
-examples_base: /usr/share/openshift/examples
+examples_base: "{% if is_atomic %}{{ openshift.common.config_base }}{% else %}/usr/share/openshift{% endif %}/examples"
```

Comment:
I'd prefer to avoid jinja conditionals if at all possible, the following should provide the same result:
```
examples_base: "{{ (openshift.common.config_base ~ '/examples') if is_atomic else '/usr/share/openshift/examples' }}"
```